services/filestash/docker-compose.yml

```diff
@@ -21,7 +21,11 @@ services:
         - traefik.http.routers.filestash.middlewares=ops_whitelist_ips@docker
       resources:
         limits:
-          memory: 512M
+          memory: 120M
+          cpus: "0.01"
```

Comment:
while I can't comment on memory usage, the CPU for sure is very low. This one will go through a lot of files and will not respond at all in time.

Reply:
raised